dbt/macros/slugify.sql

```diff
@@ -1,13 +1,13 @@
--- Variation of dbt_utils.slugify macro using kebab-case instead of snake_case
-{% macro kebab_slugify(string) %}
+-- Variation of dbt_utils.slugify macro using strict snake_case
+{% macro slugify(string) %}
     {#- Lower case the string -#}
     {% set string = string | lower %}
 
-    {#- Replace spaces, slashes, and underscores with hyphens -#}
-    {% set string = modules.re.sub("[ _/]+", "-", string) %}
+    {#- Replace spaces, slashes, and hyphens with underscores -#}
+    {% set string = modules.re.sub("[ /-]+", "_", string) %}
```

Comment:
Note that the order of this regex is now super important. The hyphen _must_ be the last character.

Reply:
**[Suggestion, non-blocking]** Sounds tricky! Might actually be worth preserving as a comment in the code?